spark-doris-connector/spark-doris-connector-base/src/main/java/org/apache/doris/spark/config/DorisOptions.java

```diff
@@ -118,6 +118,8 @@ public class DorisOptions {
 
     public static final ConfigOption<String> READ_MODE = ConfigOptions.name("doris.read.mode").stringType().defaultValue("thrift").withDescription("");
 
+    public static final ConfigOption<String> DORIS_READ_FLIGHT_SQL_PREFIX = ConfigOptions.name("doris.read.arrow-flight-sql.prefix").stringType().defaultValue("ApplicationName=Spark ArrowFlightSQL Query").withDescription("");
+
```

Comment:
In what scenarios do you need to customize this prefix? Different Spark programs?

Reply:
Via spec prefix correctly, when a Spark reader process causes high load on Doris, Doris admin can quick locate the spark app with the prefix